app/views/blocks/register.pug

```diff
@@ -0,0 +1,20 @@
+section.register
+  .container
+    .register__breadcrumb 
+      +formHeading({class1: 'register__breadcrumb-home', title: 'Trang chủ / ', desc: 'Giới thiệu'})
+    .register__heading
+      .register__heading__inner
+        +formHeading({class1: 'register__heading-title', title: 'Đăng ký', src: '../../images/titleleft-dark.png'})
+      a.btn.btn-active.register__heading-btn(href="#")  Đăng nhập
+    form.form(action="#")
+      h3.form-title Thông tin cá nhân
+      +formGroup({label: 'Tên trước*'})
+      +formGroup({label: 'Tên sau*'})
+      +formGroup({label: 'Email*'})
+      +formGroup({type: 'password', label: 'Password*'})
```

Comment:
Bổ sung thêm id cho form group nhé
https://stackoverflow.com/questions/36334658/label-for-and-input-id

Reply:
oke ạ